addons/stock_account/models/product.py

```diff
@@ -381,6 +381,11 @@ def _run_fifo_vacuum(self, company=None):
             }
             vacuum_svl = self.env['stock.valuation.layer'].sudo().create(vals)
 
+            # If some negative stock were fixed, we need to recompute the standard price.
```

Comment:
I'm not sure I get how negative stock was fixed when we arrive here.

Reply:
because previously you have
``` python
            if svl_to_vacuum.currency_id.is_zero(corrected_value):
                continue
```

if you get here, an estimated price was fixed